database/migrations/2020_11_06_175320_add_drop_foreign_on_author_book_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddDropForeignOnAuthorBookTable extends Migration
```

Comment:
Why creating separate migration for 'foreign keys'? It would be hard to maintain.

Reply:
could't write the drop code on down method. I have take this step.

Reply:
Done. foreign key drop added on there specific table migration file.